src/ui/pages/WalletSelect/WalletSelect.tsx

```diff
@@ -26,6 +84,15 @@ export function WalletSelect() {
     queryFn: () => walletPort.request('uiGetWalletGroups'),
     useErrorBoundary: true,
   });
+
+  const ownedAddressesCount = useMemo(
```

Comment:
It feels like we can get the list of addresses here and pass it to the <PortfolioRow/> to avoid 2 almost similar useMemo